### DISCUSSION TITLE
Allow tagging entire feeds

### DISCUSSION THREAD

**Discussion by SmutnyStefan** [tughi/aggregator-android#53] [Ideas]
Now I don't see any way to categorize them

**Comment by tughi** [comment]
For reading purposes you can use tags to group entries from specific feeds via tag rules.

Grouping feeds for managing purposes is not supported.

**Reply by SmutnyStefan** [reply]
Yes I know I can tag entries, but I would like to tag feeds ( by subject, theme or whatever) which is useful when someone has dozens of feeds. Tagging entries is imho not much of a help, since to tag an entry one needs to read it in the first place - but i use tags to find the needed material, not to read it again latter

**Reply by SmutnyStefan** [reply]
By the way - i don’t think it is worth a separate issue, but what would you think about coloured tags? I saw it someplace and I think the idea is nice. For some people it is helpful to associate a term or concept with a colour, thus it helps them memorize things a bit easier

**Reply by tughi** [reply]
I agree... I was even thinking of letting users assign custom icons
